Made some of the edits you recommended, but wasn't able to run that command on account of  

```
Running machine...

Unexpected error occurred:
 Error: Cannot find module 'machine'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/Alex/Documents/machinepack-mandrill/index.js:2:18)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

which also happened when I tried to send a plaintext email.

--- a/machines/send-template-email.js
+++ b/machines/send-template-email.js
@@ -1,7 +1,7 @@
 module.exports = {
 
-  friendlyName: 'Send email (plaintext)',
-  description: 'Send a plaintext email to the specified recipient.',
+  friendlyName: 'Send email (template)',
+  description: 'Send a templated email to the specified recipient.',
   cacheable: false,
 
   inputs: {

--- a/machines/send-template-email.js
+++ b/machines/send-template-email.js
@@ -37,9 +37,8 @@ module.exports = {
     },
     templateContent: {
       friendlyName: 'Data',
-      description: "The data object you want to inject into the template.",
-      example: "{'name': 'Jane'}",
-      typeclass: "dictionary"
+      description: "An array of data you want to inject into the template.",
+      example: "[{'name': 'of attribute', 'content': 'of attribute'}, {'name': 'of other attribute', 'content': 'of other attribute'}]"
     },
     message: {
       friendlyName: 'Message',
@@ -81,7 +80,7 @@ module.exports = {
       form: {
         key: inputs.apiKey,
         template_name: inputs.templateName,
-        template_content: JSON.stringify(inputs.templateContent),
+        template_content: inputs.templateContent,
         message: {
           to: [{
             email: inputs.toEmail,
@@ -97,6 +96,7 @@ module.exports = {
       },
       json: true
     }, function(err, response, httpBody) {
+
       if (err) {
         return exits.error(err);
       } else if (response.status >= 300 || response.status < 200) {

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
     "machineDir": "machines",
     "machines": [
       "send-plaintext-email",
+      "send-template-email",
       "add-template",
       "list-templates",
       "delete-template",

